docs/siem/detections/api/lists/api-get-list-items.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[lists-api-get-item]]
+=== Get list item
+
+Retrieves a list item using its `id`, or its `list_id` and `value` fields.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/lists/items`
+
+===== URL query parameters
+
+The URL query must include one of the following:
+
+* `id` - `GET /api/lists/items?id=<id>`
+* `list_id` and `value` - `GET /api/lists/items?list_id=<list_id>&value=<value>`
+
+===== Example requests
+
+Retrieves the list item with an `id` of `internal-ip-address`:
+
+[source,console]
+--------------------------------------------------
+GET api/lists/items?id=internal-ip-excludes
+--------------------------------------------------
+// KIBANA
+
+Retrieves the list item in the `internal-ip-excludes` container, with a `value`
+of `10.0.0.1`:
+
+[source,console]
+--------------------------------------------------
+GET api/lists/items?list_id=internal-ip-excludes&value=10.0.0.1
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+==== Response payload
+
+[source,json]
+--------------------------------------------------
+{
+  "created_at": "2020-07-07T04:10:26.733Z",
+  "created_by": "Threat Hunter",
+  "id": "internal-ip-address",
+  "list_id": "internal-ip-excludes",
+  "tie_breaker_id": "c5a7088e-173c-4c6c-be50-801d25e9d10a",
+  "type": "ip",
+  "updated_at": "2020-07-07T04:10:26.733Z",
+  "updated_by": "Threat Hunter",
+  "value": "10.0.0.1"
+}
```

Comment:
So a GET won't return the `_version` where POST will return the `_version`.

Example:

```ts
❯ ./post_list_item.sh
{
  "_version": "WzUyMjMwMTUsMV0=",
  "id": "ip_item",
  "type": "ip",
  "value": "127.0.0.1",
  "created_at": "2020-08-04T21:04:11.940Z",
  "created_by": "yo",
  "list_id": "ip_list",
  "tie_breaker_id": "ce78d9d0-3dfe-49db-8718-31fe3182cd71",
  "updated_at": "2020-08-04T21:04:11.940Z",
  "updated_by": "yo"
}
```

compared to GET:
```ts
❯ ./get_list_item_by_value.sh ips.txt 127.0.0.1
[
  {
    "created_at": "2020-08-04T20:35:55.194Z",
    "created_by": "yo",
    "id": "mbwwu3MBM_3LKPKZDKzB",
    "list_id": "ips.txt",
    "tie_breaker_id": "47d3d57b-c5fe-4948-8d73-d147cfcf5c22",
    "type": "ip",
    "updated_at": "2020-08-04T20:35:55.194Z",
    "updated_by": "yo",
    "value": "127.0.0.1"
  }
]
```

I would document things accurately here and later when I fix bugs or expose missing pieces I will update you to update the docs.

However, as I illustrated in the previous examples I think it's important that we should that the GET will return an array of items (up to a 10k limit) and the reasoning is that you can have a list of IP's and be doing a CIDR query or you could have a list of ip_range and your IP you query for will intersect with multiple ip_range's.